lib_neutral_prompt/cfg_denoiser_hijack.py

```diff
@@ -240,58 +241,53 @@ def forge_sample(self, denoiser_params, cond_scale, cond_composition, original_f
             return original_function(self, denoiser_params, cond_scale, cond_composition)
 
         self.inner_model.inner_model.forge_objects.unet.model_options['cond_composition'] = cond_composition
+        self.inner_model.inner_model.forge_objects.unet.model_options['uncond'] = cond_from_a1111_to_patched_ldm(denoiser_params.text_uncond)
+
         return original_function(self, denoiser_params, cond_scale, cond_composition)
 
-    @ldm_patched_modules_hijacker.hijack('samplers')
+
+    @samplers_hijacker.hijack('calc_cond_uncond_batch')
     def calc_cond_uncond_batch(model, cond, uncond, x_in, timestep, model_options, original_function):
         if not global_state.is_enabled:
             return original_function(model, cond, uncond, x_in, timestep, model_options)
 
         cond_composition = model_options['cond_composition']
+        uncond = model_options['uncond']
```

Comment:
Note that it is necessary to override `uncond` here, as the `uncond` parameter will be `None` if cfg scale is 1. We need uncond in any case to do calculations in delta space.